app/assets/src/components/ui/icons/InfoIconSmall.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SmallInfoIcon = props => {
```

Comment:
Thank you for taking the opportunity to start updating names according to our new naming guidelines! 💯